scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py

```diff
@@ -136,8 +136,10 @@ def submit_eqtl_jobs(
                 )
 
                 keys = os.path.join(input_path, 'OneK1K_CPG_IDs.tsv')
+                version = output_dir.split('/')[-1]
```

Comment:
An alternative approach would be to not launch this script with the output path containing the cell type, but instead rely on the script to append the cell type to the path itself when you write outputs. Does that make sense?

Reply:
Yes, that's a good idea. And would look a lot cleaner so I could just use `output_path` instead